src/Microsoft.AspNetCore.Mvc/Microsoft.AspNetCore.Mvc.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <Description>ASP.NET Core MVC is a web framework that gives you a powerful, patterns-based way to build dynamic websites and web APIs. ASP.NET Core MVC enables a clean separation of concerns and gives you full control over markup.</Description>
-    <TargetFramework>netstandard2.0</TargetFramework>
+    <TargetFramework>netcoreapp3.0</TargetFramework>
```

Comment:
Would you need to update downstream consumers first - e.g. https://github.com/aspnet/Identity/blob/master/src/UI/Microsoft.AspNetCore.Identity.UI.csproj#L31?

Reply:
Already in progress :) https://github.com/aspnet/Identity/pull/2072. I'm starting at the top of the graph, working down.